src/L6/datasorter_test.py

```diff
@@ -1,12 +1,34 @@
 import unittest
+import os
 
-from L6.datasorter import DataSorter
```

Comment:
Were you able to run the Python tests after removing the declaration with the `L6` package? I haven't tested this code myself, but I don't know if the IDE or the script would complain of not being able to find both DataSorter and NotCSVException because both classes are technically inside of the `L6.datasorter` module.

Reply:
It gave me issues until I removed this line. that's why I changed how the import is, tell me if this could be an issue.

Reply:
It could, yes, most likely what happened is that you set your working directory in PyCharm to L6 instead of src/. The working directory should be src/ so that you can access all of the packages in the project.

For this particular assignment it won't be a big deal, but we'll have two sets of instructions for the teacher, one for running my previous assignments in the repo and one for L6. My personal recommendation is to keep the fully qualified path to the module (i.e., including the package name) so that the convention is consistent across the repository. If you have questions on how to set the working directory in PyCharm, hit me up.